Snabble/UI/Payment/QRCheckoutViewController.swift

```diff
@@ -22,29 +22,110 @@ final class QRCheckoutViewController: UIViewController {
     private let shop: Shop
     weak var delegate: PaymentDelegate?
 
+    private var customLabel: UILabel {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .label
+        label.textAlignment = .center
+        label.setContentHuggingPriority(.defaultLow + 1, for: .horizontal)
+        label.setContentHuggingPriority(.defaultLow + 1, for: .vertical)
+        return label
+    }
+
     public init(shop: Shop,
                 checkoutProcess: CheckoutProcess,
                 cart: ShoppingCart) {
         self.shop = shop
         self.cart = cart
         self.process = checkoutProcess
 
-        super.init(nibName: nil, bundle: SnabbleSDKBundle.main)
-
-        self.title = L10n.Snabble.QRCode.title
+        super.init(nibName: nil, bundle: nil)
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
+    override public func loadView() {
+        let contentView = UIView(frame: UIScreen.main.bounds)
+        contentView.backgroundColor = .systemBackground
+
+        let checkoutIdLabel = customLabel
+        checkoutIdLabel.font = UIFont.systemFont(ofSize: 13)
+
+        let totalPriceLabel = customLabel
+        totalPriceLabel.font = UIFont.systemFont(ofSize: 17, weight: .medium)
+
+        let explanationUpperLabel = customLabel
+        explanationUpperLabel.font = UIFont.systemFont(ofSize: 17, weight: .light)
+        explanationUpperLabel.lineBreakMode = .byWordWrapping
+        explanationUpperLabel.numberOfLines = 0
+
+        let qrCodeView = UIImageView()
+        qrCodeView.translatesAutoresizingMaskIntoConstraints = false
+        qrCodeView.contentMode = .scaleToFill
+        qrCodeView.setContentHuggingPriority(.defaultLow + 1, for: .horizontal)
+        qrCodeView.setContentHuggingPriority(.defaultLow + 1, for: .vertical)
+
+        let explanationBottomLabel = customLabel
+        explanationUpperLabel.font = UIFont.systemFont(ofSize: 11)
+        explanationBottomLabel.numberOfLines = 0
+
+        let cancelButton = UIButton(type: .system)
+        cancelButton.translatesAutoresizingMaskIntoConstraints = false
+        cancelButton.setTitle(L10n.Snabble.cancel, for: .normal)
+        cancelButton.titleLabel?.font = UIFont.systemFont(ofSize: 15)
+        cancelButton.setTitleColor(SnabbleUI.appearance.accentColor, for: .normal)
+        cancelButton.isEnabled = true
+        cancelButton.isUserInteractionEnabled = true
+        cancelButton.addTarget(self, action: #selector(cancelButtonTapped(_:)), for: .touchUpInside)
```

Comment:
Was the look of the button like that before and not like the other buttons? You can use `makeSnabbleButton()` to see the other style, but you have to remove line 77 otherwise the tintColor doesn't work.

Reply:
Fixed bottom button layout looking like EmbeddedCodeVC, as there is no layout at Sketch for this screen. Red color is an accent for chosen shop:
![image](https://user-images.githubusercontent.com/40164951/165961818-82b179ad-f31f-466d-bc65-39894e9ebf66.png)